ui/src/app/applications/components/applications-list/applications-list.tsx

```diff
@@ -515,6 +516,18 @@ export const ApplicationsList = (props: RouteComponentProps<{}>) => {
                                                                                 </h5>
                                                                             </EmptyState>
                                                                         )}
+                                                                        sortOptions={[
+                                                                            {title: 'Name', compare: (a, b) => a.metadata.name.localeCompare(b.metadata.name)},
+                                                                            {
+                                                                                title: 'Created At',
+                                                                                compare: (a, b) => a.metadata.creationTimestamp.localeCompare(b.metadata.creationTimestamp)
+                                                                            },
+                                                                            {
+                                                                                title: 'Synchronized',
```

Comment:
Do you want to change `Synchronized` to `App Health`?

Reply:
No, I would keep Synchronized as it is (it's the last time the sync operation finished - either successfully or not).

But an `App Health` sort order option makes sense as well: show Unknown, Degraded, Out of sync, Synced in this order!

Reply:
updated